src/models/sequelize.ts

```diff
@@ -2,9 +2,19 @@ import { Sequelize } from "sequelize";
 import dotenv from "dotenv";
 dotenv.config();
 
-const DB_URL = process.env.DB_URL || "postgres://localhost:5432/slack";
+const DB_DB = process.env.DB_DB;
+const DB_HOST = process.env.DB_HOST;
+const DB_USERNAME = process.env.DB_USERNAME;
+const DB_PASSWORD = process.env.DB_PASSWORD;
 
-const sequelize = new Sequelize(DB_URL);
+if (!DB_DB || !DB_HOST || !DB_USERNAME || !DB_PASSWORD) {
+  throw new Error('Missing required environment variables for database connection.');
+}
+
+const sequelize = new Sequelize(DB_DB, DB_USERNAME, DB_PASSWORD, {
+  host: DB_HOST,
+  dialect: 'postgres',
+});
 
 const testDbConnection = async (sequelize: Sequelize) => {
   try {
```

Comment:
just updated the db connection according to the docs so it worked with docker (you can ignore this bit if you want)